models/events.py

```diff
@@ -132,6 +137,7 @@ class EventQueryResponse(BaseModel):
     max_capacity: int
     public: bool
     attending: List[user_models.UserId]
+    comment_ids: List[str]
```

Comment:
Is the plan to eventually create a some sort of comment object? Asking out of curiosity

Reply:
Yep, actually, the code is already done (look for `feedback` modules).